### ISSUE TITLE
wont let me add any more loading screens....

### ISSUE THREAD

**Issue by casigus** [TheGameCreators/GameGuruRepo#4609]
![image](https://user-images.githubusercontent.com/68471279/236084244-f984e0f1-ecab-4a66-a176-861300fbbc12.png)

**Comment by davetgc** [comment]
I've marked this as a high bug, as although it's not a bug as such, it's obviously stopping you progressing.

**Comment by casigus** [comment]
yeah! I literally cant finish the game because of this aha! its just sorta shows a blank screen now inbetween levels which isn't great.

**Comment by LeeBamberTGC** [comment]
@casigus The cap is 99 screens per game project, do you want to use more than 99 screens?

**Comment by casigus** [comment]
99! daym! I dont think I was using that many in titanic!!!
________________________________
